### DISCUSSION TITLE
Direct Windows .bat Installation

### DISCUSSION THREAD

**Discussion by PoyBoi** [PoyBoi/StableDiffusionHelper#3] [Announcements]
# What's New ?

1. Added GUI support
2. Cleaner Installer batch file
3. Updated code file structure
4. Check readme file for more details

<hr /><em>This discussion was created from the release <a href='https://github.com/PoyBoi/StableDiffusionHelper/releases/tag/v1.0-GUI'>Direct Windows .bat Installation</a>.</em>

**Comment by ride5k** [comment]
hi, i am attempting to install and run your package, but i do not see a "run.bat" to launch after installation.

i followed the steps which instructed me to download "install.bat" and run, which completed successfully.

there are three batch files created in the stablediffusionhelper subfolder (gui.bat, ipynb.bat, and troubleshoot.bat) however after running each of them the files complete downloading/updating and then exit.

**Reply by PoyBoi** [reply]
Hey @ride5k, sorry to hear that you're having an issue, I have re-posted this as an issue in "Issues" and I think you would have received a notification for the same, I'd love to know more about your issue there 😊

[Here's the link to the opened issue](https://github.com/PoyBoi/StableDiffusionHelper/issues/5), please do let me know more about your issue there, I'd be happy to help you there :D

**Reply by imago10** [reply]
> Hey @ride5k, sorry to hear that you're having an issue, I have re-posted this as an issue in "Issues" and I think you would have received a notification for the same, I'd love to know more about your issue there 😊
> 
> [Here's the link to the opened issue](https://github.com/PoyBoi/StableDiffusionHelper/issues/5), please do let me know more about your issue there, I'd be happy to help you there :D

I have same issue, how to solve it?
